src/components/benchmarks/BenchmarkAssignmentModal.tsx

```diff
@@ -173,6 +183,12 @@ export const BenchmarkAssignmentModal = (
             />
           </Box>
         )}
+
+        {errorMessage && (
+          <Box className={$benchmark.benchmarkDescriptionBox}>
+            {errorMessage}
```

Comment:
Output the error here for now, in next PR we will aim to prevent this from happening